recipe/yum_requirements.txt

```diff
@@ -0,0 +1,5 @@
+java-1.8.0-openjdk-devel
+unzip
+zip
+zlib-devel
```

Comment:
Don't hold the recipe b/c of this! I am just curious if `zlib` needs any changes to be fit for bazel.